src/realm/sync/tools/hist_command.cpp

```diff
@@ -2242,7 +2242,7 @@ int main(int argc, char* argv[])
     Group group{realm_path, encryption_key_3}; // Throws
     using gf = _impl::GroupFriend;
     int file_format_version = gf::get_file_format_version(group);
-    if (file_format_version != 23) {
+    if (file_format_version != 24) {
```

Comment:
This seems correct for the current file format version, but could we call this [Group::read_only_version_check()](https://github.com/realm/realm-core/blob/9fea3082183d7d46cf83c03100aed9c00cdf1cec/src/realm/group.hpp#L506) to check if the file format is valid to read rather than checking a hardcoded version?